### PR TITLE
Added a new roundend sound

### PR DESCRIPTION
**Pull request by Jujumatic**
I have immortalised pyko in roundend sounds

:cl:
add: Added Pyko to roundend sounds
/:cl:

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Review by praisenarsie — DISMISSED** [review]

**Review by KayeArray — DISMISSED** [review]

**Review by GuyonBroadway — DISMISSED** [review]

File: hippiestation/code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -24,7 +24,8 @@
 		'hippiestation/sound/roundend/nitrogen.ogg',
 		'hippiestation/sound/roundend/henderson.ogg',
 		'hippiestation/sound/roundend/gameoverinsertfourcoinstoplayagain.ogg',
-		'hippiestation/sound/roundend/reasonsunknown.ogg'\
+		'hippiestation/sound/roundend/reasonsunknown.ogg'
```
Comment (ghost) [inline hippiestation/code/controllers/subsystem/ticker.dm]: where is my comma....

**Review by ghost — DISMISSED** [review]
i don't compile so good...

**Review by praisenarsie — APPROVED** [review]

**Comment by Pyko1** [comment]
sure, why not

**Comment by praisenarsie** [comment]
it was already +2 you mong!!!

**Pull request merged**